packages/cli/lib/developerTestAccountCreate.js

```diff
@@ -0,0 +1,186 @@
+const SpinniesManager = require('./ui/SpinniesManager');
+const {
+  getAccountId,
+  accountNameExistsInConfig,
+  updateAccountConfig,
+  writeConfig,
+} = require('@hubspot/local-dev-lib/config');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const {
+  createDeveloperTestAccount,
+} = require('@hubspot/local-dev-lib/developerTestAccounts');
+const { i18n } = require('./lang');
+const {
+  debugErrorAndContext,
+  logErrorInstance,
+} = require('./errorHandlers/standardErrors');
+const {
+  isMissingScopeError,
+  isSpecifiedError,
+} = require('@hubspot/local-dev-lib/errors/apiErrors');
+const {
+  getAccessToken,
+  updateConfigWithAccessToken,
+} = require('@hubspot/local-dev-lib/personalAccessKey');
+const { getHubSpotWebsiteOrigin } = require('@hubspot/local-dev-lib/urls');
+const { uiAccountDescription } = require('./ui');
+const {
+  personalAccessKeyPrompt,
+} = require('./prompts/personalAccessKeyPrompt');
+const { enterAccountNamePrompt } = require('./prompts/enterAccountNamePrompt');
+
```

Comment:
Also not a blocker, but looks like there's a lot of very similar logic between this file and lib/sandboxCreate.js . Maybe there's a way to consolidate somehow?

Reply:
yeah I had considered this but felt it might have been too many changes at once, I'll prob elect to break it out into a separate PR and consolidate a bunch of things

Reply:
Yeah that makes sense 👌